string marker_name
float64 new_velocity                  
float64 velocity_change
geometry_msgs/Quaternion new_direction
geometry_msgs/Quaternion direction_change
